pom.xml

```diff
@@ -402,7 +408,7 @@
             <dependency>
                 <groupId>com.google.guava</groupId>
                 <artifactId>guava</artifactId>
-                <version>24.1.1-jre</version>
+                <version>27.0.1-jre</version>
```

Comment:
To elaborate, by controlling it here you be able to avoid excluding this and commons-text in bunch of places above since the management version will override everything